services/reference/_partials/filter-methods/_eth_getfilterchanges-returns.mdx

```diff
@@ -1,4 +1,4 @@
-- `log object array`: (array) An array of log objects, or an empty array if nothing has changed since last poll.
+- `log object array`: (array) An array of log objects, or an empty array if nothing has changed since the last poll.
```

Comment:
Can you change other instances of this for consistency? For example, the `eth_getLogs` return value.